Exercise_4.py

```diff
@@ -1,23 +1,44 @@
 # Python program to insert element in binary tree  
+# Time Complexity : O(height)
```

Comment:
O(h) for inorder.
O(n^2) for Insert, the usage of a queue instead would bring the time complexity down to O(n).

Reply:
Noted.